tasks/task585_preposition_text_generation.json

```diff
@@ -0,0 +1,11472 @@
+{
+    "Contributors": [
+        "Ishan Purohit"
+    ],
+    "Source": [
+        "Prepositional Paraprasing (Dataset - https://www.cse.iitb.ac.in/~girishp/nc-dataset/) (Paper - https://aclanthology.org/2020.findings-emnlp.386.pdf)"
+    ],
+    "Categories": [
+        "Text Generation"
+    ],
+    "Definition": "In this task, you're given inputs of two words and your task is to predict the correct preposition that connects these two words. This involves rewriting the noun compound as a paraphrase that conveys its meaning explicitly, e.g., orange juice: \u201cjuice made from the orange\u201d or \u201cjuice with orange flavor\u201d.",
```

Comment:
Drop the Unicode characters in the definition. (e.g., `\u201c`). Also, list the possible outputs (`of`, etc).

Reply:
> Drop the Unicode characters in the definition. (e.g., `\u201c`). Also, list the possible outputs (`of`, etc).

But in the dataset, only one output is given.

Reply:
Maybe I didn't say it right; in the description state what are the possible outputs ("of", "with", etc). Does that make sense?